### PR TITLE
Riddl and clean protobuf gen

### PR DESCRIPTION
**Pull request by reid-spencer**
This should give the template's design module a working version of sbt-riddl, 0.27.0

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -16,23 +17,23 @@ lazy val root = project
     publishTo := Some(Resolver.defaultLocal)
   )
   .aggregate(
-    //  design,
+    design,
+    common,
+    utils,
+    service1,
+    service2,
```
Comment (AlexWeinstein92) [inline build.sbt:24]: common, utils, service1, and service2 are already brought in via gateway and its dependency tree - so do we really need to aggregate them here?

File: common/src/main/protobuf/com/example/common/kalix_policy.proto [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// This is the default Access Control List (ACL) for all components of this Kalix Service
+syntax = "proto3";
+
+package com.ossum.amenities.account;
```
Comment (AlexWeinstein92) [inline common/src/main/protobuf/com/example/common/kalix_policy.proto:4]: this doesn't seem right

File: common/src/main/protobuf/com/example/common/nothing.proto [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// this file only exists to generate something kalixy
```
Comment (AlexWeinstein92) [inline common/src/main/protobuf/com/example/common/nothing.proto:1]: We already have `api/nothing_service`, so either that or this can be deleted

**Comment by AlexWeinstein92** [comment]
Both the proto files included here already exist in `improving.template.common`. It would likely be better to rename that to `example.common` than to do what you have done re-creating these files, as the re-creations declare `package` paths using `ossum.amenities.account`

File: build.sbt (hunk 1 above)
Comment (reid-spencer) [reply build.sbt:24]: I think so, build dependencies only apply when doing build things (compile, test) but if you want to apply non-build things across everything then you need to aggregate them.

File: common/src/main/protobuf/com/example/common/kalix_policy.proto (hunk 2 above)
Comment (reid-spencer) [reply common/src/main/protobuf/com/example/common/kalix_policy.proto:4]: Remember that common doesn't actually need to define a service or have a policy but we do want to do kalixy kinds of things in common. In order to make it not fail on build, you need to have this policy file in common.  Without it the build fails. c'est la vie.

Comment (AlexWeinstein92) [reply common/src/main/protobuf/com/example/common/kalix_policy.proto:4]: I was referring specifically to the package name. I don't think we want `ossum.amenities.account`

File: build.sbt (hunk 1 above)
Comment (AlexWeinstein92) [reply build.sbt:24]: Fair enough

**Comment by reid-spencer** [comment]
@AlexWeinstein92 - Riddl and common both now work with these recent changes. Please re-review.

File: common/src/main/protobuf/com/example/common/kalix_policy.proto (hunk 2 above)
Comment (reid-spencer) [reply common/src/main/protobuf/com/example/common/kalix_policy.proto]: Will fix.

Comment (reid-spencer) [reply common/src/main/protobuf/com/example/common/kalix_policy.proto]: Actually, it is already fixed on the new commits

**Pull request merged**